C-Plus-Plus/coinchange(Dynamic prog).cpp

```diff
@@ -0,0 +1,47 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+    long long int sum,n,i,j,k;
+    cin>>sum>>n;
+    long long int a[n];
+    for(i=0;i<n;i++)
+    {
+        cin>>a[i];
+    }
+    long long int b[n+1][sum+1];
+    for(i=0;i<=n;i++)
+    {
+        b[i][0]=1;
+    }
+    for(i=0;i<=n;i++)
+    {
+        if(i==0)
+        {
+        for(j=1;j<sum+1;j++)
+        {
+            b[0][j]=0;
+        }
+        }
+        else
+        {
+            for(j=1;j<sum+1;j++)
+            {
+                if(j<a[i-1])
+                {
+                    b[i][j]=b[i-1][j];
+                }
+                else
+                {
+                    b[i][j]=b[i-1][j]+b[i][j-a[i-1]];
+                }
+            }
+        }
+    }
+    cout<<b[n][sum];
+}
+
```

Comment:
Add SPACE & TIME Complexities.

Reply:
Plus you've to update the README.md file.

Reply:
Okay.Thanks!

Reply:
Hey @lostgirljourney , Please check it out.I have updated README.md and gave the code link at the bottom and also added space and time complexities in the cpp file.Please let me know if there are further changes to do.Thank you!